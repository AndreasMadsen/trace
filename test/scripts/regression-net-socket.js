'use strict';

const net = require('net');

const server = net.createServer(function (socket) {
  throw new Error('trace');
});

server.listen(0, 'localhost', function () {
  const addr = server.address();
  const socket = net.connect(addr.port, addr.address, function () {
    // just hang, it will throw
  });
});
