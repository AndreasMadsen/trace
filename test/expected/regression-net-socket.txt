/scripts/regression-net-socket.js:6
  throw new Error('trace');
  ^

Error: trace
    at Server.<anonymous> (/scripts/regression-net-socket.js:6:9)
    at Object.<anonymous> (/scripts/regression-net-socket.js:9:8)
